server/src/config-helper.ts

```diff
@@ -3,6 +3,10 @@ import { S3, SSM } from 'aws-sdk';
 import { SESClientConfig } from '@aws-sdk/client-ses';
 import { config } from 'dotenv';
 
+import Logger from './lib/logger';
```

Comment:
Since we're likely to make use of Logger a lot, and it all generally relies on the same logic, I think we can make Logger a singleton. This would mean there'd only be on instance of Logger regardless of how many times its referenced (lowering memory usage). And to use it, call a static method (e.g. `Logger.log(name:string, log:Log)`). Or we could create new classes that extend the Log class (e.g. `AuthRouterLogger extends Logger`) centrally, in a new `/loggers` directory if we anticipate using the same log instance across files. That would be a bit of memory overhead as well, but allows Loggers to be shared. 

If we go one of the aforementioned routes, and it's a lot of work, I can create a ticket and make the suggested changes when theres time and keep this PR the same so we can ship it sooner